### DISCUSSION TITLE
Our meshing software (MeshIt) is now available from git

### DISCUSSION THREAD

**Discussion by mcacace** [idaholab/moose#16537] [Q&A Meshing] [answered]
Foreword: This email would be of interest for the (small) group of users dealing with fracture/reservoir THM applications.
Dear Moose community,
I would like to share that out in-house meshing software is now available from github (https://github.com/bloech/MeshIt). The main scope of MeshIt is to provide Boundary conforming Delaunay 3D meshes with a specific target to complex fractured reservoirs. It provide the flexibility to integrate 2D and 1D lower dimensional elements in a format that is compatible with the framework standard via a dedicated exodus output. You can find an application on the Moose website where we have coupled MeshIt to one of our Moose based application.  Some information (non exhaustive) on the theory behind the software can be found in a relative old paper at https://doi.org/10.1007/s12665-015-4537-x. 
Hope this could be of any help to the community and we are glad to accept any commit/questions/... they should arise,
Mauro

**Comment by GiudGiud** [comment, marked as answer]
Just marking this as answered. It ll remain visible in the meshing Q&A section.
Thanks for sharing.
